omnibus/config/software/datadog-agent.rb

```diff
@@ -68,7 +69,8 @@
 
   ## build the custom action library required for the install
   if windows?
-    command "invoke customaction.build"
+    platform = windows_arch_i386? ? "x86" : "x64"
```

Comment:
`platform` is already defined on line 47 so there's no need to redefine it here (`if` branches do not have their own scope in ruby)
But I understand if you want to keep this re-definition since it's not obvious that `platform` is still defined here.

Reply:
The reason I did it that way is variable scope; I wasn't sure if the previous definition would be maintained outside of the `if windows?` block

Reply:
👍Having some re-definition (that doesn't cost much performance-wise) to avoid  wondering later how this variable was defined (thanks to ruby shenanigans) is fine